i3ipc.py

```diff
@@ -413,6 +413,15 @@ def __init__(self, data, parent, conn):
             else:
                 setattr(self, attr, None)
 
+        # Make sure marks is never None (use []) to simplify client code. Older
+        # versions of i3 used mark instead of marks, so collapse that into
+        # marks to provide a consistent API.
+        if not self.marks:
```

Comment:
`self.mark` (the original) is set to `None` by the above code and not deleted in the new version of the ipc. The Con should rather not have the property at all in this case.